Source/SwiftLintFramework/Rules/Lint/YodaConditionRule.swift

```diff
@@ -1,41 +1,15 @@
-import Foundation
 import SourceKittenFramework
+import SwiftSyntax
 
-public struct YodaConditionRule: ASTRule, OptInRule, ConfigurationProviderRule {
+public struct YodaConditionRule: OptInRule, ConfigurationProviderRule {
```

Comment:
No longer makes any SourceKit requests:

```suggestion
public struct YodaConditionRule: OptInRule, ConfigurationProviderRule, SourceKitFreeRule {
```